tests/golden_tests/desugar_file/main_aux.bend

```diff
@@ -0,0 +1,9 @@
+def main:
+  y = 89
+  def aux(x):
```

Comment:
Can we have an example with multiple local defs of same nesting depth? With equal and different names.
Also try calling them to see if the correct name is being used when we shadow one local def with another.